src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/Collect.cs

```diff
@@ -0,0 +1,412 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.PowerFx.Core.App.ErrorContainers;
+using Microsoft.PowerFx.Core.Binding;
+using Microsoft.PowerFx.Core.Entities;
+using Microsoft.PowerFx.Core.Errors;
+using Microsoft.PowerFx.Core.Functions;
+using Microsoft.PowerFx.Core.Functions.FunctionArgValidators;
+using Microsoft.PowerFx.Core.Localization;
+using Microsoft.PowerFx.Core.Types;
+using Microsoft.PowerFx.Core.Utils;
+using Microsoft.PowerFx.Syntax;
+using Microsoft.PowerFx.Types;
+
+namespace Microsoft.PowerFx.Core.Texl.Builtins
+{
+    // Collect(collection:*[...], item1:![...]|*[...], ...)
+    internal class CollectFunction : BuiltinFunction
```

Comment:
>CollectFunction [](http://example.com/codeflow?start=19&length=15)

Assuming this is coming from PA? Where there any interesting changes?

Reply:
The interesting part were to merge PFx/PA business logic and some commented blocks are PA exclusive I'll need to review them along with the PA team.